jujugui/static/gui/src/app/components/inspector-relations/test-inspector-relations.js

```diff
@@ -32,7 +32,8 @@ describe('InspectorRelations', function() {
   });
 
   it('can render the onboarding', function() {
-    var getRelationDataForService = sinon.stub.returns([]);
+    var getRelationDataForService = sinon.stub();
+    getRelationDataForService.returns([]);
```

Comment:
Oh, it was `sinon.stub.returns([])` when it should have been `sinon.stub().returns([])`